uint8 status
Header header
float64 time
float64[7] X
float64[7] Xn
float64[2] uMPC
float64[2] uCBF
float64[2] uTot
float64 flagQP
float64 V
float64 h
